### ISSUE TITLE
header&cookies的bug

### ISSUE THREAD

**Issue by flc1125** [jkniest/HTMLCache#15]
虽然没有使用，但看代码感觉会有bug。当使用`header`或`cookie`时候，不会输出`header`和`cookie`

-------------

**如下：**

```php
response(view(''))->header('foo', 'bar')->cookie('test', 'test', 10);
```

**Comment by jkniest** [comment]
I'm sorry but I only speak english and german. But I guess, based on google translator and your code example, that the cache package ignores cookies and header fields when caching.

In this case this would be true. Currently the only cache parameter which determinate how the page is cached are the url, the status code and the error / validation bag.

I will think about including header- and cookie fields into this list. So that the same page does cache differently based on the header fields.

If I understood you wrong, please correct me :-)

Have a nice day,
Jordan

**Comment by flc1125** [comment]
你的理解完全正确。希望以后增加支持此类的功能。

祝你幸运！

**Comment by jkniest** [comment]
I have looked into it a little bit and I have implemented the header caching. So header fields are now also cached and returned. So in your example the `->header('foo', 'bar')` would work.

But I don't plan to include cookies. I personally don't see any real value in caching cookies. If you have any idea feel free to write :-)

I will test it a little bit more and release a new version later that day.
